### ISSUE TITLE
obtaining integer CN estimates from non-diploid chromosomes 

### ISSUE THREAD

**Issue by 7ehuang** [abyzovlab/CNVpytor#208]
Thank you for the active maintenance of this software! I read in issue [#81 ](https://github.com/abyzovlab/CNVpytor/issues/81) and [#88](https://github.com/abyzovlab/CNVpytor/issues/88) that I can obtain copy number estimates by multiplying the "CNV level" column by 2 and rounding to the nearest integer. However, would this still be the case when analyzing data from cells with aneuploidy (i.e. K562, which is predominantly triploid)? Because then in these cases, the majority of the genome is not necessarily diploid. 

I ask because I am comparing my CNVpytor calls with the results from this [paper](https://genome.cshlp.org/content/29/3/472), and the CN calls are highly discordant.

**Comment by arpanda** [comment]
We have an example for the K562 in the cnvpytor manuscript (supplymentary figure S1). In that figure, the normalization is applied based on the bimodal hypothesis (sum of two gaussion distributions with ratio of mean values equal to 2, related to copy numbers 2 and 4 ). 

The implementation is hard coded and let you know once we add a user option to enable it through command line.

If you want to unlock option by changing the code, you can remove `False and ` in line of file `root.py`, under implementation of the caller function you are using :
```
bim = fit_bimodal(bins[:-1], hist)
if False and bim is not None:
```

Thank you,
Arijit

**Comment by 7ehuang** [comment]
> We have an example for the K562 in the cnvpytor manuscript (supplymentary figure S1). In that figure, the normalization is applied based on the bimodal hypothesis (sum of two gaussion distributions with ratio of mean values equal to 2, related to copy numbers 2 and 4 ).
> 
> The implementation is hard coded and let you know once we add a user option to enable it through command line.
> 
> If you want to unlock option by changing the code, you can remove `False and ` in line of file `root.py`, under implementation of the caller function you are using :
> 
> ```
> bim = fit_bimodal(bins[:-1], hist)
> if False and bim is not None:
> ```
> 
> Thank you, Arijit

Thank you for your reply! Does this mean that by default (i.e. without making the changes to the code that you suggested), the normalization is not based on the bimodal hypothesis? 

Just to clarify, are the results from the manuscript based on the default mean-shift caller or the joint caller?

**Comment by arpanda** [comment]
Yes, the default is not set for the bimodal hypothesis as you can see the `false` is hard coded in the code.

The mean shift caller has the limitation that its based on read depth only. For the supplementary figure S1, the normalization is based on joint caller. 

-Arijit
